README.md

```diff
@@ -40,9 +50,10 @@ This template comes with a basic project structure that you may find useful:
 | [`src/dev_tools.rs`](./src/dev_tools.rs) | Dev tools for dev builds                              |
 
 Feel free to move things around however you want, though.
+The best way to get started is to just play around with what you find in [`src/game/`](./src/game).
```

Comment:
```suggestion
The best way to get started is to play around with what you find in [`src/game/`](./src/game).
```
I think I would also move this line to before the table, but not sure.

Reply:
[`7552039` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/7552039db72db62518b502ca9da16997c1cb4f52)

Reply:
The location is resolved, suggestion to remove `just` not yet resolved.

Reply:
Ah, thanks for the reminder [`dedd01c` (#158)](https://github.com/TheBevyFlock/bevy_quickstart/pull/158/commits/dedd01cef41ce323c500b2fa0d4c004a181bc107)